src/QsCompiler/DataStructures/SyntaxTokens.fs

```diff
@@ -130,33 +137,61 @@ type LambdaKind =
     | Operation
 
 /// A lambda expression.
-type 'expr Lambda =
+type Lambda<'Expr, 'Type> =
     private
         {
             kind: LambdaKind
-            param: QsSymbol
-            body: 'expr
+            argTuple: QsTuple<LocalVariableDeclaration<QsLocalSymbol, 'Type>>
```

Comment:
nit: can we you "parameter" language for definitions and "argument" language for calls? This could be called `paramTuple`.

Reply:
Thanks! As an aside, would we be open to having a wider change like that, changing to using "ParameterTuple" throughout the syntax tree, in a future PR? I ask because that seems mildly disruptive for 3rd party devs, but we are already being disruptive in that same way in this PR, so I thought that this might be a good place to do that, to avoid churn. What's your recommendation for these kinds of changes? Should we try to lump as many breaking changes together when we need to make a breaking change, or should we have more frequent, but less impactful breaking changes?